From dbd4801acac74c875a8a3d3b6f7e232e9ee62c87 Mon Sep 17 00:00:00 2001
From: Yuya Hamamachi <yuya.hamamachi.sx@renesas.com>
Date: Fri, 4 Mar 2022 15:47:03 +0900
Subject: [PATCH 1/3] script/python/starterkit.py: Fix timeout when using M3v1
 + mot mode.

Flash-writer requires 2nd_Y in mainloop regardless of the board type.
The case of needless 2nd_Y is only M3 v1 starter kit + normal mode.
So, combination M3 v1 and mot mode occurs timeout.
This patch fixes this timeout issue.

Signed-off-by: Yuya Hamamachi <yuya.hamamachi.sx@renesas.com>
---
 script/python/starterkit.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/script/python/starterkit.py b/script/python/starterkit.py
index 825b63b..b5659f5 100755
--- a/script/python/starterkit.py
+++ b/script/python/starterkit.py
@@ -112,6 +112,7 @@ class rom_write_guide(base.guide):
     def guide_for_mot(self):
         mot_file = "{}/starterkit/config/mot".format(self.top())
         cpld_cmd = self.ttm_array(mot_file, "cpld_cmd")
+        self.__use_2nd_Y = True
 
         # power on
         # and stop the U-Boot autorun
-- 
2.31.1

